### PR TITLE
update license comments to include a copyright

### PR DESCRIPTION
**Pull request by elmiko**
this change just adds some copyright information to be in compliance with the apache license.

### CODE REVIEW

**Comment by elmiko** [comment]
if, for whatever reason, we don't want to assign the copyright to red hat, an alternative would be to credit the copyright to the "rebasebot authors" and then make an authors file in the root.

File: rebasebot/github.py [hunk 1]
```diff
@@ -1,3 +1,16 @@
+#    Copyright 2022 Red Hat, Inc.
```
Comment (lobziik) [inline rebasebot/github.py:1]: Dunno if it's important or not, but this was added in 2023

**Review by lobziik — COMMENTED** [review]
files in /tests was added in 2023 as well, I don't know if it really matters.

lgtm

won't label it right now to give room for other opinions.

File: rebasebot/github.py (hunk 1 above)
Comment (elmiko) [reply rebasebot/github.py:1]: i'll update the tests files and this one, thanks!

**Comment by elmiko** [comment]
updated with 2023 copyrights

**Review by lobziik — COMMENTED** [review]
/lgtm

**Pull request merged**